app/Services/UserService.php

```diff
@@ -0,0 +1,73 @@
+<?php
+
+namespace App\Services;
+
+use App\Models\Role;
+use App\Models\User;
+use Auth;
+
+class UserService
+{
+    /**
+     * @return bool|array
+     */
+    public function checkAccess()
+    {
+        if (Auth::guard('api')->user()->id != request()->route('record')->getAttributes()['id']) {
+            $err['forbidden'] =  'You don\'t have to access this resource.';
```

Comment:
Maybe something like that: "You don't have permission to access this resource" ?

Reply:
ok